404.txt

```diff
@@ -89,7 +89,7 @@ into some framework" type of topics.
   PR229 The cosmic workshop for kids .......................... @justinasjaronis
   PR230 Vizualūs Efektai Pradedantiesiems .............................. @jurgis
   PR231 Technologijų karas ........................................ @gediminasgu
-
+  PR236 ką gali pasigaminti savo kojų wattais.........................@vytautasj
```

Comment:
```suggestion
  PR236 ką gali pasigaminti savo kojų wattais........................ @vytautasj
```

Reply:
The [`format_topic` function](https://github.com/ntacamp/404.notrollsallowed.com/blob/master/bin/update.sh#L36) solves that automatically.